awx/main/migrations/0118_v400_execution_environments.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 2.2.11 on 2020-06-29 15:47
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import taggit.managers
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('taggit', '0003_taggeditem_add_unique_index'),
+        ('main', '0117_v400_remove_cloudforms_inventory'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='unifiedjob',
+            name='pull',
+            field=models.BooleanField(default=True),
+        ),
+        migrations.AddField(
+            model_name='unifiedjobtemplate',
+            name='pull',
+            field=models.BooleanField(default=True),
+        ),
+        migrations.CreateModel(
+            name='ExecutionEnvironment',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', models.DateTimeField(default=None, editable=False)),
+                ('modified', models.DateTimeField(default=None, editable=False)),
+                ('description', models.TextField(blank=True, default='')),
+                ('name', models.CharField(max_length=512)),
+                ('url', models.CharField(help_text='The registry location where the container is stored.', max_length=1024, verbose_name='URL')),
```

Comment:
I'm wondering if it makes sense to have `name` and `url` be 2 separate things, or if there can just be one `name` field. What do you think?

Reply:
You could be right, though I'm not sure about the 'name' field being the one that survives.  I'd think that if we kept only one that it should be the url field.

Reply:
Mostly because they are technically referred to as "fully qualified image names"

Reply:
hm, ok